pkg/kubelet/cm/cpumanager/policy_options.go

```diff
@@ -19,13 +19,41 @@ package cpumanager
 import (
 	"fmt"
 	"strconv"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	kubefeatures "k8s.io/kubernetes/pkg/features"
 )
 
 const (
 	// FullPCPUsOnlyOption is the name of the CPU Manager policy option
 	FullPCPUsOnlyOption string = "full-pcpus-only"
 )
 
+var (
+	stableOptions = sets.NewString()
+	betaOptions   = sets.NewString(
+		FullPCPUsOnlyOption,
+	)
+	alphaOptions = sets.NewString()
```

Comment:
can we order these alpha, beta, stable, instead of stable, beta, alpha?

Reply:
Sure. My intention was to have at glance the "most important" options first, but in retrospect the "most important", meaning the one the reader should pay most attention to, are the *less* stable rather than the other way around.